client/my-sites/design-menu/design-tools/index.js

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
> I'd rather we just had completely disconnected components.

I totally agree, but in order to show something in the `DesignMenu`, somewhere we have to write down a component and give it props. I'm open to other ideas as to where that could be.